main.css

```diff
@@ -0,0 +1,199 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300..700&display=swap");
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: Poppins, sans-serif;
+  display: grid;
+  place-items: center;
+  height: 100vh;
+  background: #e6e6e6;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 327px;
+  margin-block: 2rem;
+  gap: 2rem;
+}
```

Comment:
- [ ] Kindly use the `grid` property for the main container of this project as it is the requirement